shared/settings/notifications/render.tsx

```diff
@@ -49,56 +56,89 @@ const Group = (props: {
             subscribed={s.subscribed}
           />
         ))}
-    </Box>
+    </Kb.Box>
     {props.unsub && (
-      <Box style={{...globalStyles.flexBoxColumn}}>
-        <Text type="BodyBig">Or:</Text>
-        <Checkbox
-          style={{marginTop: globalMargins.small}}
+      <Kb.Box style={{...Styles.globalStyles.flexBoxColumn}}>
+        <Kb.Text type="BodySmall">Or</Kb.Text>
+        <Kb.Checkbox
+          style={{marginTop: Styles.globalMargins.xtiny}}
           onCheck={props.onToggleUnsubscribeAll || null}
           disabled={!props.allowEdit}
           checked={!!props.unsubscribedFromAll}
           label={`Unsubscribe me from all ${props.unsub} notifications`}
         />
-      </Box>
+      </Kb.Box>
     )}
-  </Box>
+    <Kb.Divider style={styles.divider} />
+  </Kb.Box2>
+)
+const EmailSection = (props: Props) => (
```

Comment:
I used to do a lot of these too, but I think if you use it as a function instead of component, it's slightly cheaper. Not a big deal though and I'm OK as-is!

Reply:
Hmm, how much cheaper? This definitely feels like a legibility win to me.

Reply:
prefer components to functions, don't optimize early for no reason